mcf-browser/src/back-end/parse-tmcf.ts

```diff
@@ -174,6 +185,128 @@ class ParseTmcf {
     return mcfLines.join('\n');
   }
 
+  /**
+   * Convert a single row to an object containining its facet
+   * and its corresponding value and date
+   * @param {string} template The string representation of a tmcf file.
+   * @param {CsvRow} row The json representation of the csv row.
+   *     Each CsvRow element of the array represents one row of the csv.
+   * @return {ParsedCsvRow | null} an object containing the facet, date,
+   *     and value
+   */
+  getFacetAndValueFromRow(template: string, row: CsvRow)
+  : ParsedCsvRow | null {
+    type Properties = {
```

Comment:
nit: usually we want to define types globally (ie. outside of a function)

Reply:
done!